### ISSUE TITLE
can not work with spring.redis.redisson.config

### ISSUE THREAD

**Issue by Dengyren** [ulisesbocchio/jasypt-spring-boot#333]
![image](https://user-images.githubusercontent.com/38429401/185112702-55217e33-4853-4025-a3d0-a6c3987e838e.png)
![image](https://user-images.githubusercontent.com/38429401/185112786-57c9a1e8-9f56-4e97-9c1a-4354323a0cee.png)
![image](https://user-images.githubusercontent.com/38429401/185113064-1ec3405f-7f30-4c33-9b63-c67051a5b8af.png)

**Comment by ulisesbocchio** [comment]
that's interesting. I'm not familiar with that kind of "YAML inlining inside YAML???" Why is `spring.redisson.config` a text value?

**Comment by ghuan** [comment]
Yes, the creation of a RedissonClient requires configuration information that is not based on the Spring configuration environment.
![image](https://github.com/ulisesbocchio/jasypt-spring-boot/assets/15809971/41866479-4721-45f7-92b7-f51f38cfd89e)
Currently, the approach would be to add a custom RedissonAutoConfigurationCustomizer before creating the RedissonClient. In this customizer, you can access the already created Config object, decrypt its properties, and replace it.
![image](https://github.com/ulisesbocchio/jasypt-spring-boot/assets/15809971/c7af6ae4-bbca-4b42-aaa1-36132d9a11dd)
